apps/google-analytics-4/lambda/serverless.yml

```diff
@@ -18,7 +18,10 @@ functions:
   api:
     environment:
       SIGNING_SECRET: ${self:custom.environment.SIGNING_SECRET}
+      SENTRY_DSN: ${self:custom.environment.SENTRY_DSN}
       STAGE: ${opt:stage, 'dev'}
+      NODE_ENV: ${env:NODE_ENV}
+      RELEASE_VERSION: ${env:CIRCLE_SHA1}
```

Comment:
TODO:

- [ ] configure `CIRCLE_SHA1` environment variable in development using dotenv (`.env` file)

Reply:
Let's discuss that TODO. @mgoudy91 and I ended up removing `dotenv` because we ended up with two many systems that were managing per environment data. It probably makes sense to move these to the serverless environment configs.

But CIRCLE_SHA1 does present some difficulties with that idea. 🤔

Reply:
Should we set a default fallback value here? Maybe something explicit like 'undefined' or 'development' -- or even fall back to NODE_ENV? That way if we see it in the logs we know where it came from as opposed to something being busted with our whole configuration

Reply:
updated to:

```
      NODE_ENV: ${env:NODE_ENV, 'development'}
      RELEASE_VERSION: ${env:CIRCLE_SHA1, 'undefined'}
```

Reply:
Re: removing dotenv – I'm fine with that, I can just add the CIRCLE_SHA1 to the npm scripts commands

Reply:
```
"scripts": {
    "debug": "NODE_ENV=development CIRCLE_SHA1=$(git rev-parse --short HEAD) SLS_DEBUG=* node inspect ./node_modules/serverless/bin/serverless.js offline",
    "build": "NODE_ENV=production CIRCLE_SHA1=$(git rev-parse --short HEAD) rimraf ./build && tsc",
    "lint": "eslint .",
    "start:dev": "NODE_ENV=development CIRCLE_SHA1=$(git rev-parse --short HEAD) nodemon",
    "test": "NODE_ENV=test CIRCLE_SHA1=$(git rev-parse --short HEAD) TS_NODE_TRANSPILE_ONLY=1 mocha",
    "test:debug": "NODE_ENV=test CIRCLE_SHA1=$(git rev-parse --short HEAD) TS_NODE_TRANSPILE_ONLY=1 mocha -- --inspect --inspect-brk",
    "test:watch": "NODE_ENV=test CIRCLE_SHA1=$(git rev-parse --short HEAD) TS_NODE_TRANSPILE_ONLY=1 mocha --watch --watch-files src --watch-files test",
    "deploy:test": "NODE_ENV=test CIRCLE_SHA1=$(git rev-parse --short HEAD) sls deploy --stage test",
    "deploy": "CIRCLE_SHA1=$(git rev-parse --short HEAD) sls deploy --stage $STAGE"
  },
```